librad/src/profile.rs

```diff
@@ -219,15 +231,4 @@ mod test {
         let id = load_profile_id(tempdir.path()).unwrap();
         assert_eq!(id, "foo");
     }
-
-    #[test]
-    fn empty_profile_file() {
```

Comment:
Wait, is there a test for the empty file causing an error?

Reply:
Added in https://github.com/radicle-dev/radicle-link/pull/496/commits/d33410cc76657db918e3ae259ee00623acaf1741